fabric-common-tags-api-v1/src/main/java/net/fabricmc/fabric/api/tags/v1/CommonBiomeTags.java

```diff
@@ -0,0 +1,63 @@
+package net.fabricmc.fabric.api.tags.v1;
+
+import net.minecraft.tag.TagKey;
+import net.minecraft.world.biome.Biome;
+
+import net.fabricmc.fabric.impl.v1.TagRegistration;
```

Comment:
```suggestion
import net.fabricmc.fabric.impl.tag.common.TagRegistration;
```

Impl package isn't versioned.